packages/flame/lib/src/extensions/rect.dart

```diff
@@ -52,6 +53,19 @@ extension RectExtension on Rect {
     ];
   }
 
+  /// Transform Rect using the transformation defined by [matrix].
+  ///
+  /// **Note:** Rotation matrices will increase the size of the [Rect] but they
+  /// will not rotate it as [Rect] does not have any rotational values.
+  ///
+  /// **Note:** Only non-negative scale transforms are allowed, if a negative
+  /// scale is applied it will return a zero-based [Rect].
+  Rect transform(Matrix4 matrix) {
```

Comment:
It should probably be mentioned that it transforms from the center, since no optional anchor was added.

Reply:
I don't think anchor would do anything here. If the transform is only limited to translations and scales, then it does not depend on what the anchor is.

Reply:
Added a comment for it regardless, but yea adding an anchor isn't possible unless we go for polygons in the future